src/sdk/pynni/nni/curvefitting_assessor/model_factory.py

```diff
@@ -110,7 +109,8 @@ def filter_curve(self):
         std = np.std(predict_data)
         for model in tmp_model:
             y = self.predict_y(model, self.target_pos)
-            if y < median + 3 * std and y > median - 3 * std:
+            epsilon = self.point_num / 10 * std
```

Comment:
Is it possible that std is zero?     ( For example,  np.std([5,5] is zero)
If yes,  you can add a very small number to 10*std.